app/Filament/Resources/UserResource/Pages/EditProfile.php

```diff
@@ -113,6 +115,28 @@ protected function getForms(): array
                                             ->options(fn (User $user) => $user->getAvailableLanguages()),
                                     ]),
 
+                                Tab::make('OAuth')
+                                    ->icon('tabler-brand-oauth')
+                                    ->schema(function () {
+                                        $providers = [];
+
+                                        foreach (config('auth.oauth') as $name => $data) {
+                                            if (!$data['enabled']) {
+                                                continue;
+                                            }
+
+                                            $unlink = array_key_exists($name, $this->getUser()->oauth);
+
+                                            $providers[] = Action::make("oauth_$name")
+                                                ->label(($unlink ? 'Unlink ' : 'Link ') . Str::title($name))
+                                                ->icon($unlink ? 'tabler-unlink' : 'tabler-link')
+                                                ->color($data['color'])
+                                                ->url('/account/oauth/' . ($unlink ? 'unlink' : 'link') .  "/$name");
```

Comment:
Should be able to do something like this:

```suggestion
                                                ->url(route('account.oauth.' . ($unlink ? 'unlink' : 'link'), ['driver' => $name]));
```